pom.xml

```diff
@@ -150,15 +150,16 @@
 					<version>3.2.0</version>
 				</plugin>
 
-				<plugin>
+					<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-checkstyle-plugin</artifactId>
-					<version>${checkstyle.plugin.version}</version>
```

Comment:
I already used the eclipse, spring boot (based on eclipse), and the s.o terminal. My s.o is windows 10, my computer has 2tb, 24gb ram, and the processor is an  i7 4720hq. 
I'm trying to use opennlp to treat ner in portuguese brazilian. Then of course perform a simple test on the tool.